.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Describe the Problem and Solution
+<!-- Write 2-3 sentences summarising the reasons behind this change (this might be the problem you're solving, or the context around the request) and the solution you have chosen. !-->  
+
+## What makes this change safe?
+<!-- DELETE WHEN COMPLETED
```

Comment:
MD comments may not actually show up on the PR, if you click "view file" on this document they are hidden. It would be ideal if we could test this prior to merging.

Reply:
We could us <# As the syntax?

Reply:
It's semi-common I think to put the instructions in comments and then expect folks to delete the comments when they replace it with their own content, but if we think that's too complex then I'd say we should just leave it as normal text in the template.

Reply:
Well yes but in this case the comment won't be visible to the change author. It'll look fairly odd.

Reply:
What do you mean? 
When creating a PR, you see something like this.

![image](https://github.com/user-attachments/assets/67d70dc3-0f87-4bd3-ab60-6aabb1bd7997)

And the expectation is that you replace everything including the comment markers ie `<!-- ... -->` so that it becomes normal text.

Reply:
I was expecting the PR to get created and the comments to be removed with time I guess? You could do it that way.

Reply:
I see, yeah, either way we should just pick a thing and train folks that's how we do it.